src/OpenTelemetry/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Notes](../../RELEASENOTES.md).
 
 ## Unreleased
 
+* Updated `OpenTelemetry.Metrics.MetricStreamIdentity` Metric tags are now considered identifiers,
```

Comment:
```suggestion
* Update Metric tags to be identifying for the Meter, aligning with the specification. Previously, Metric tags were omitted from being identifying.
```

Reply:
Will anyone reading this understand it? 🤣

How about...

* [Meter.Tags](https://learn.microsoft.com/dotnet/api/system.diagnostics.metrics.meter.tags) will now be considered when resolving the SDK metric to update when measurements are recorded. Meters with the same name and different tags will now lead to unique metrics.

PS: There is a lint error can't merge until that is resolved.

Reply:
Fine with the latter, made the change. can linter re instigate, I will check if its yet too long.